### PR TITLE
Fix merging of summaries to match the tutorial "TensorFlow Mechanics 101"

### PR DESCRIPTION
**Pull request by nikikilbertus**
The example does not work with `tf.summary.merge_all()`. Changing it to `tf.merge_all_summaries()` like in the [tutorial](https://www.tensorflow.org/versions/r0.11/tutorials/mnist/tf/index.html#train-loop) fixes the issue.

### CODE REVIEW

**Comment by nikikilbertus** [comment]
I signed it.

**Review by drpngx — APPROVED** [review]

**Comment by zheng-xq** [comment]
@tensorflow-jenkins test this please

File: tensorflow/models/image/cifar10/cifar10_eval.py [hunk 1]
```diff
@@ -81,36 +81,36 @@ def eval_once(saver, summary_writer, top_k_op, summary_op):
       print('No checkpoint file found')
       return
 
-    # Start the queue runners.
```
Comment (drpngx) [inline tensorflow/models/image/cifar10/cifar10_eval.py]: That code block is not reachable since it's after `return`?

**Review by drpngx — CHANGES_REQUESTED** [review]

File: tensorflow/models/image/cifar10/cifar10_eval.py (hunk 1 above)
Comment (nikikilbertus) [inline tensorflow/models/image/cifar10/cifar10_eval.py]: Sorry, my bad. I had it dedented in my fork and then accidentally indented one level too much.

**Comment by danmane** [comment]
I made this change originally. Using tf.summary.merge_all() is definitely intended, as for tf1.0 we are moving all the summary ops into the tf.summary namespace. I will dig into why the example is broken and fix it while keeping usage of tf.summary.merge_all()

**Pull request closed without merging**